### PR TITLE
Coverage report jenkins fix

### PR DESCRIPTION
**Pull request by k80bowman**
## Description
The code coverage report json that we were uploading to dev separately is being deleted when Jenkins deploys to dev. We need to include it as part of the build tar in order to make sure it doesn't get deleted.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: jenkins/common.groovy [hunk 1]
```diff
@@ -115,6 +115,10 @@ def archive(dockerContainer, String ref, String envName) {
     withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'vetsgov-website-builds-s3-upload',
                      usernameVariable: 'AWS_ACCESS_KEY', passwordVariable: 'AWS_SECRET_KEY']]) {
       sh "echo \"${buildDetails}\" > /application/build/${envName}/BUILD.txt"
+
+      sh "cd /application && yarn test:coverage-only"
+      sh "node script/app-coverage-report.js"
```
Comment (ndsprinkle) [inline jenkins/common.groovy:120]: The app-coverage-report script doesn't copy to the build directory anymore, if that was the expected behavior here. It's in /coverage, so it'll need to be copied from there or modify the app-coverage-report script again. One more example of us coming full circle with this ticket haha.

Comment (k80bowman) [reply jenkins/common.groovy:120]: Good catch, thank you!

Comment (k80bowman) [reply jenkins/common.groovy:120]: Done!

**Comment by k80bowman** [comment]
Closing this. Opted to use a static file for now. We will come back to potentially automating the unit test coverage report another time.

**Pull request closed without merging**